build.cake

```diff
@@ -81,7 +81,8 @@ Task("Cover")
         if (DirectoryExists("coverage"))
             CleanDirectories("coverage");
     })
-    .Does(DotCover("net452", RunTests(unitTests, "net452")))
+    .Does(RunTests(unitTests, "net452", "Coverage=Skip"))
+    .Does(DotCover("net452", RunTests(unitTests, "net452", "Category!=explicit & Coverage!=Skip")))
```

Comment:
Excluding a handful more slowest tests (>10 seconds duration) shaves off 20 minutes from the AppVeyor run and only 0,12% coverage.